src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform 300ms, color 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
+      cursor: pointer;
+    }
   }
 
   &__card:not(:nth-child(4n)) {
```

Comment:
[SASS] - Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.

Reply:
thanks for checking, I have missed this rule.